src/kernel-install/kernel-install.in

```diff
@@ -297,6 +297,14 @@ trap cleanup EXIT
 
 KERNEL_INSTALL_STAGING_AREA="$(mktemp -d -t kernel-install.staging.XXXXXXX)"
 
+export KERNEL_INSTALL_IMAGE_TYPE=""
+if [ -f "$1" ]; then
+    KERNEL_INSTALL_IMAGE_TYPE="$(bootctl kernel-identify "$1")"
```

Comment:
Needs an entry in the man page.

Reply:
Sure, I'll add then when we have agreement on the code changes.